src/components/App.tsx

```diff
@@ -505,12 +494,13 @@ class App extends React.Component<ExcalidrawProps, AppState> {
       this.setState({ isLoading: true });
     }
 
-    let scene = await loadScene(null, null, this.props.initialData);
+    const { initialData } = this.props;
```

Comment:
is that really needed?

Reply:
are you referring to destructuring? I did it coz it's used twice so destructured here.

Reply:
I think not worth it for two references..